pyproject.toml

```diff
@@ -625,6 +621,9 @@ unfixable = [
 "tests/**" = [
   "S101", # Allow 'assert' in tests
 ]
+"src/meltano/migrations/versions/**" = [
+  "N806", # Allow uppercase variable names
+]
```

Comment:
I think this is covered by the `exclude` setting?

```suggestion
```

Reply:
Yep.